frontend/src/data/variables/AcsHealthInsuranceProvider.ts

```diff
@@ -55,37 +54,59 @@ class AcsHealthInsuranceProvider extends VariableProvider {
 
     if (breakdowns.geography === "national") {
       df = df
-        .pivot(breakdowns.demographicBreakdowns.race_and_ethnicity.columnName, {
-          fips: (series) => USA_FIPS,
-          fips_name: (series) => USA_DISPLAY_NAME,
+        .pivot(
+          [
+            breakdowns.demographicBreakdowns.race_and_ethnicity.columnName,
+            breakdowns.demographicBreakdowns.sex.columnName,
+          ],
+          {
+            fips: (series) => USA_FIPS,
+            fips_name: (series) => USA_DISPLAY_NAME,
+            with_health_insurance: (series) => series.sum(),
+            without_health_insurance: (series) => series.sum(),
+            total_health_insurance: (series) => series.sum(),
+          }
+        )
+        .resetIndex();
+    }
+
+    if (!breakdowns.demographicBreakdowns.race_and_ethnicity.enabled) {
+      df = df.pivot(
+        ["fips", "fips_name", "sex"].concat(
+          breakdowns.hasOnlySex() ? [] : ["age"]
+        ),
+        {
+          race: (series) => ALL_RACES_DISPLAY_NAME,
```

Comment:
Since we're only looking at one breakdown at a time, we can just drop the other columns here and only consider the relevant one. They would get dropped later in removeUnrequestedColumns() anyway

You should be able to pivot to get totals just once by passing in a variable as the column name (race_and_ethnicity or sex) by putting it in [], see example: https://github.com/SatcherInstitute/health-equity-tracker/blob/e94ae9693654d84fec6f4f6b6b853479c4d5de08/frontend/src/data/variables/VariableProvider.ts#L55

You could even extract this function into computeTotalsForField(breakdownColumnName:string, fieldsToTotal: string[]) and reuse with AcsProvider